src/components/timebank/timebank-daterange-picker.tsx

```diff
@@ -0,0 +1,86 @@
+import { DatePicker } from "@mui/x-date-pickers";
+import { DateTime } from "luxon";
+import { useEffect, useState } from "react";
+import { DailyEntry } from "../../generated/client";
+import { Range } from "../../types"
+import strings from "../../localization/strings";
+
+/**
+ * Component properties
+ */
+interface Props {
+  dailyEntries: DailyEntry[];
+  setSelectedEntries: (selectedEntries: DailyEntry[]) => void;
+  today: DateTime;
+}
+
+/**
+ * Date Range Picker component associated with multi bar chart.
+ *
+ * @param props Component properties
+ * @returns Two date pickers enabling the selection of date range.
+ */
+const DateRangePicker = (props: Props) => {
+  const { setSelectedEntries, dailyEntries, today } = props;
+
+  const [range, setRange] = useState<Range>({
+    start: today.minus({ days: 7 }),
+    end: today
+  });
+
+  useEffect(() => {
+    getDateRangeEntries();
+  }, [range]);
+
+  /**
+   * Runs a for-loop that goes through the selected days, then sets entries from those days into a state to be displayed in the multi bar chart.
```

Comment:
This description can be shortened, e.g. we can see form the code there is a for-loop so no need to say in in the comment. It could be something like: Gets daily entries within time range.